### PR TITLE
Add release notes for 3.9.0

### PR DESCRIPTION
**Pull request by garethbowen**
# Description

medic/cht-core#6312

# Code review items

- Readable: Concise, well named, follows the [style guide](https://github.com/medic/medic-docs/blob/master/development/style-guide.md), documented if necessary.
- Documented: Configuration and user documentation on [medic-docs](https://github.com/medic/medic-docs/)
- Tested: Unit and/or e2e where appropriate
- Internationalised: All user facing text
- Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in Changes.md.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

File: release-notes/docs/3.9.0.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+There are no required changes to the [supported software matrix](https://docs.communityhealthtoolkit.org/core/overview/supported-software/#dependencies) from `3.0.0`.
+
+| Node | CouchDB | Browsers | SMS bridge | Android | medic-android | medic-couch2pg |
+|----|----|----|----|----|----|---|
+| 8.11+ | 2.1+ | Chrome 53+, Firefox latest | medic-gateway | 4.4+ | 0.4.5+ | 3.0+ |
+
+## DHIS2 integration
+
+The CHT now supports calculating and exporting data for [DHIS2](https://www.dhis2.org) integration. The Core Framework can now be configured to calculate DHIS2 dataValues, aggregate patient level data, view aggregate data offline, and export the aggregate data formatted for DHIS2. The data is also available via an API so you can access it from an interoperability layer such as OpenHIM.
```
Comment (garethbowen) [inline release-notes/docs/3.9.0.md:38]: I'd love to link to the demo here. Can we publish [this video](https://drive.google.com/file/d/1n_5b3-AdvA0ru57q5Kt6mq5714O6DFoW/view) to YouTube for example?

File: release-notes/docs/3.9.0.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+
+For more information read [the documentation](https://docs.communityhealthtoolkit.org/apps/features/integrations/dhis2/).
+
+Issues: [cht-core#5661](https://github.com/medic/cht-core/issues/5661) and [cht-core#6002](https://github.com/medic/cht-core/issues/6002)
+
+## Supervise target progress
+
+Targets can now be configured to be aggregated and synced to allow supervisors to view the progress of their health workers towards their goals. These new "aggregate" targets show an overview of all workers and can be selected to show each worker's progress individually.
+
```
Comment (garethbowen) [inline release-notes/docs/3.9.0.md:47]: Have we got a good screenshot (or screenshots) we can include here?

**Comment by garethbowen** [comment]
@Hareet Can you please review to ensure the upgrade notes are sufficient? Ideally a community partner would be able to update without our help!

@MaxDiz Please review to capture all the product education info you want. The blog post will be based on this so now's a good time to get this sorted.

File: release-notes/docs/3.9.0.md (hunk 2 above)
Comment (MaxDiz) [reply release-notes/docs/3.9.0.md:47]: I think this should read:

...These new "aggregate" targets show an overview of all workers progress within their branch and can be selected to show each workers progress individually.

Read the [feature overview](https://docs.communityhealthtoolkit.org/apps/features/targets/#supervisor-view) and the [technical documentation](https://docs.communityhealthtoolkit.org/apps/reference/targets/#targetsjs) to see how to configure this new aggregated target.

^^ feature overview link is not live yet. @abbyad is reviewing PR

File: release-notes/docs/3.9.0.md [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+For reports that aren't patient or place centric you can now register a case with a generated `case_id` short code. This allows subsequent reports to identify which case they refer to.
+
+Read more about configuring the `accept_case_reports` transition in the [CHT documentation](https://docs.communityhealthtoolkit.org/apps/reference/app-settings/transitions/).
+
+Issues: [cht-core#6291](https://github.com/medic/cht-core/issues/6291).
+
+## Instance monitoring
+
+A [new API](https://github.com/medic/cht-core/blob/master/api/README.md#monitoring) has been added to allow tracking of a range of metrics in the CHT platform to make monitoring, alerting, and debugging of your instance easier.
```
Comment (MaxDiz) [inline release-notes/docs/3.9.0.md:70]: I think @abbyad is in the process of moving api documentation to the doc site. It's not live yet, but link should be changed before publishing.

File: release-notes/docs/3.9.0.md [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+
+Targets can now be configured to be aggregated and synced to allow supervisors to view the progress of their health workers towards their goals. These new "aggregate" targets show an overview of all workers and can be selected to show each worker's progress individually.
+
+Read [the documentation](https://docs.communityhealthtoolkit.org/apps/reference/targets/#targetsjs) to see how to configure this new aggregated target.
+
+Issues: [cht-core#4839](https://github.com/medic/cht-core/issues/4839) and [cht-core#4840](https://github.com/medic/cht-core/issues/4840)
+
+## RapidPro integration
+
+The Outbound Push feature has been extended to support submitting data to RapidPro so you can now trigger workflows based on changes in the CouchDB database.
```
Comment (MaxDiz) [inline release-notes/docs/3.9.0.md:54]: Interactive, structured messaging for health triage and clinical referrals is now possible with the integration of [RapidPro](https://rapidpro.io/). This opens new opportunities for a semi-automated, direct to patient approach to health assessments and care coordination at the community level. The Outbound Push feature has been extended to support submitting data to RapidPro so you can trigger workflows based on changes in the CouchDB database. 

Read the [feature overview](https://docs.communityhealthtoolkit.org/apps/features/integrations/rapidpro/) to learn more about this new integration.

^^placeholder link for documentation

File: release-notes/docs/3.9.0.md (hunk 3 above)
Comment (garethbowen) [reply release-notes/docs/3.9.0.md:70]: Pending merge: https://github.com/medic/cht-docs/pull/142

File: release-notes/docs/3.9.0.md (hunk 2 above)
Comment (garethbowen) [reply release-notes/docs/3.9.0.md:47]: Updated. Thanks!

Comment (MaxDiz) [reply release-notes/docs/3.9.0.md:47]: Marc noticed a mismatch on the previously attached image. Here's a better one

target-detail

File: release-notes/docs/3.9.0.md (hunk 3 above)
Comment (garethbowen) [reply release-notes/docs/3.9.0.md:72]: Updated

File: release-notes/docs/3.9.0.md (hunk 2 above)
Comment (garethbowen) [reply release-notes/docs/3.9.0.md:47]: Replaced.

File: release-notes/docs/3.9.0.md (hunk 4 above)
Comment (garethbowen) [reply release-notes/docs/3.9.0.md]: Linked to placeholder...

File: release-notes/docs/3.9.0.md (hunk 1 above)
Comment (garethbowen) [reply release-notes/docs/3.9.0.md:38]: No response...

**Pull request merged**